specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1306 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-07-20-preview",
+    "title": "azurelargeinstance",
+    "description": "The AzureLargeInstance Management client",
+    "x-ms-code-generation-settings": {
+      "name": "azurelargeinstanceClient"
```

Comment:
suggest to properly camel or pascal case instead (I forgot which is required): e.g. AzureLargeInstanceClient / azureLargeInstanceClient